pkg/ccl/utilccl/BUILD.bazel

```diff
@@ -15,6 +15,7 @@ go_library(
         "//pkg/server",
         "//pkg/settings",
         "//pkg/settings/cluster",
+        "//pkg/sql",
```

Comment:
This a hefty dependency for a util pkg; I wonder if we could move that setting registration somewhere less expensive to depend on

Reply:
do you feel this way even though the pig already depends on `server` ?

Reply:
"The pig" intrigues me :) Is that a typo or does it stand for something other than "package in question"?

Reply:
I assume autocorrect of pkg